tools/Update-Library.ps1

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+$SrcFolder = "..\src"
+$LibConfigFile = "..\library.properties"
+
+git clone https://github.com/Azure/azure-sdk-for-c sdkrepo
+
+$Files = gci -Recurse -Include *.h, *.c .\sdkrepo\sdk  | ?{ $_.DirectoryName -INOTMATCH "tests|sample" -AND $_.Name -INOTMATCH "curl|win32|az_posix" }
```

Comment:
Oh, also, do we imagine always filtering out anything other than just the Core and IoT service related files? Or would we want to try and include other service files if/when they get supported?

Reply:
The reason why I didn't do like that is because the list of files to add is much larger than the ones to filter.
I agree it would be safer, but it will become much more convoluted, and we will validate the builds every time.

We will filter only what's not wanted in the Arduino library. When new clients are added, the teams will do the work to make sure they are included.